avocado/utils/build.py

```diff
@@ -58,9 +59,37 @@ def make(path, make='make', env=None, extra_args='', ignore_status=False,
         cmd += ' %s' % makeopts
     if extra_args:
         cmd += ' %s' % extra_args
-    make_process = process.system(cmd,
-                                  env=env,
-                                  ignore_status=ignore_status,
-                                  allow_output_check=allow_output_check)
+    make_process = process.run(cmd,
+                               env=env,
+                               ignore_status=ignore_status,
+                               allow_output_check=allow_output_check)
     os.chdir(cwd)
     return make_process
+
+
+def make(path, make='make', env=None, extra_arg='', ig_status=False,
+         allow_out_check='none'):
```

Comment:
Please don't change the API, which means don't change the argument names. This is what causes the CI failures and it's essential to keep them as they were before.